query-engine/connector-test-kit/build.sbt

```diff
@@ -7,7 +7,7 @@ lazy val commonSettings = Seq(
   organization := "com.prisma",
   organizationName := "Prisma",
   scalaVersion := "2.12.7",
-  parallelExecution in Test := true,
+  parallelExecution in Test := false,
```

Comment:
You can't just flip the switch here without telling anybody. Is this intentional?

Reply:
It is not and I'm flipping it back with the tmpfs switch. Wanted to see if this makes the tests green (it did) and how slow it will be (too slow).